crates/c/src/lib.rs

```diff
@@ -1094,6 +1090,24 @@ void __wasm_export_{ns}_{snake}_dtor({ns}_{snake}_t* arg) {{
     }
 }
 
+impl<'a> InterfaceGenerator<'a> {
+    fn reosurce_handle_names(&mut self, id: TypeId, name: &str) -> (String, String) {
```

Comment:
I think this and the `match` just below may have snuck back in by accident?

Reply:
umm you're right let me fix that